src/PotentialCalculation/SuccessiveOverRelaxation/SOR.jl

```diff
@@ -71,33 +72,58 @@ Thus, there cannot be local extrema.
 """
 @inline @fastmath function handle_depletion(
     new_potential::T, 
-    point_type::PointType, 
+    imp_scale::T,
     neighbor_potentials::NTuple{6,T}, 
     q_eff_imp::T, 
     volume_weight::T,
     sor_const::T
-)::Tuple{T, PointType} where {T}
+)::Tuple{T, T} where {T}
     vmin::T = min(neighbor_potentials...)
     vmax::T = max(neighbor_potentials...)
+    
+    isLess = new_potential < vmin
```

Comment:
Do we have variables named like this (uppercase letters in between)? Why not `is_less`?

Reply:
Did not really think about that name here. I can change it.